src/handbook/development/team-npm-registry.md

```diff
@@ -0,0 +1,68 @@
+---
+navTitle: Team NPM Registry
+---
+
+# FlowFuse Team NPM Registry
+
+To enable a local development environment for the Team NPM feature follow these steps.
+
+## Registry Setup
+
+- Create a directory to hold the NPM registry 
+    ```
+    mkdir npm-registry
+    cd npm-registry
+    ```
+- Create 
+    ```
+    mkdir config storage
+    ```
+- Install verdaccio
+    ```
+    npm install --prefix=. verdaccio
+    ```
+- Clone auth plugin
+    ```
+    git clone https://github.com/FlowFuse/verdaccio-ff-auth.git
```

Comment:
As it is currently a private repo, I had to use `git clone git@github.com:FlowFuse/verdaccio-ff-auth.git`. I assume the repo will be made public - if so, it can stay as-is.

Reply:
I don't see any reason for it to stay private